cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/config-operator-role-binding/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: config-operator-role-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- kind: ServiceAccount
+  name: config-operator
+  namespace: sdap
```

Comment:
Hm.. I don't think we want to host an cluster scoped operator in a namespace we don't control. This makes the `config-operator` SA in `sdap` namespace of full control over all pods in the cluster. I don't think we should allow that, sorry.

Reply:
@tumido, I removed the spark updates from the PR.
Regarding your comment here, we don't need actually to have this operator with a cluster scope, how should I change that ?